src/styles/components/syntax/_syntax-theme.scss

```diff
@@ -0,0 +1,88 @@
+.diff {
```

Comment:
Why not using the _syntax-highlighter.scss? It is already targets to that. Theme doesn't provide any beneficial thing here. Just using the main scss would be fine.

Reply:
Good catch, forgot to clean this up actually. It was a separate file before as I just imported an existing theme file from highlight.js to experiment with.